ports/zephyr/CMakeLists.txt

```diff
@@ -18,7 +18,7 @@ add_custom_target(
   outputexports
   COMMAND echo CC="${CMAKE_C_COMPILER}"
   COMMAND echo AR="${CMAKE_AR}"
-  COMMAND echo Z_CFLAGS=${system_includes} ${includes} ${definitions} ${options}
+  COMMAND echo Z_CFLAGS=${includes} ${definitions} ${options} ${system_includes}
```

Comment:
This should not be needed.
Fixed the issue here: https://github.com/zephyrproject-rtos/zephyr/pull/28584
that could result in an empty `-isystem`.